dev-utils/dep-info.js

```diff
@@ -56,32 +56,37 @@ function generateDependencyInfo (deps, modulesPath) {
   return allLicenses
 }
 
-function generateNotice (rootDir) {
-  if (!rootDir) rootDir = './'
-  const { dependencies, name } = JSON.parse(
-    readFileSync(path.join(rootDir, './package.json'), 'utf8')
-  )
-  var depInfo = generateDependencyInfo(
-    Object.keys(dependencies),
-    path.join(rootDir, './node_modules')
-  )
-  var allLicenses = `
+function generateNotice() {
+  const packagesDir = join(__dirname, '../packages')
```

Comment:
Why not accept a parameter here instead? IMO, it's more usable if it has parameter instead of being hardcoded!

Reply:
Sure, just thought this script wont be used anywhere.